openy_af4_vue_app/src/components/Results.vue

```diff
@@ -20,14 +20,14 @@
         </div>
         <div v-else :class="resultsClasses">
           <div class="header d-md-none hidden-md hidden-lg">
-            <span class="results">
+            <span class="results" v-if="data.count">
```

Comment:
This fixed a prior JS error thrown when trying to run `formatPlural` on an empty string.